app/containers/Header/SearchForm.jsx

```diff
@@ -28,6 +30,15 @@ const SearchForm = ({ placeholder, className, onBlur, searchFormId }) => {
     [text, searchFormId, onBlur],
   );
 
+  if (
+    createdHistory.location.pathname.includes('/search') === false &&
```

Comment:
`includes` already returns boolean. Can be just `!createdHistory.location.pathname.includes('/search')`.